### PR TITLE
feat: add docs about multiple Cypress image

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/executor-cypress.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ describe('The Home Page', () => {
 ## **Creating the Testkube Test Script**
 
 Create the Testkube test script from this example. The parameters passed are **repository**, **branch** and **the path where the project exists**. In the case of a mono repository, the parameters are **name** and **type**.
+We will use default Cypress executor (Testkube Cypress image).
```
Comment (jfermi) [inline docs/executor-cypress.md:32]:
```suggestion
We will use the default Cypress executor (Testkube Cypress image).
```

File: docs/executor-cypress.md [hunk 2]
```diff
@@ -273,6 +274,46 @@ $ kubectl testkube get execution 615d5372b046f8fbd3d955d2
 Test execution completed in 1m45.405939s
 ```
 
+## Using different Cypress images
```
Comment (jfermi) [inline docs/executor-cypress.md:277]:
```suggestion
## Using Different Cypress Images 
```

File: docs/executor-cypress.md [hunk 3]
```diff
@@ -273,6 +274,46 @@ $ kubectl testkube get execution 615d5372b046f8fbd3d955d2
 Test execution completed in 1m45.405939s
 ```
 
+## Using different Cypress images 
+
+In the Cypress world, usually you want to have control over your Runtime environment. Testkube can easily handle that for you!
```
Comment (jfermi) [inline docs/executor-cypress.md:279]:
```suggestion
In the Cypress world, there are instances when you want to have control over your Runtime environment. Testkube can easily handle that for you! 
```

File: docs/executor-cypress.md [hunk 4]
```diff
@@ -273,6 +274,46 @@ $ kubectl testkube get execution 615d5372b046f8fbd3d955d2
 Test execution completed in 1m45.405939s
 ```
 
+## Using different Cypress images 
+
+In the Cypress world, usually you want to have control over your Runtime environment. Testkube can easily handle that for you! 
+We're building several Cypress images to handle features that different versions of Cypress can support.
+
+To use a different executor you can use one of our pre-built ones (for Cypress 8, 9, 10 and Custom Testkube imgae) or build your own Docker image based on a Cypress executor.
```
Comment (jfermi) [inline docs/executor-cypress.md:282]:
```suggestion
To use a different executor you can use one of our pre-built ones (for Cypress 8, 9, 10 and Custom Testkube images) or build your own Docker image based on a Cypress executor.
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

File: docs/docs/4-test-types/executor-cypress.md [hunk 5]
```diff
@@ -277,6 +278,46 @@ $ kubectl testkube get execution 615d5372b046f8fbd3d955d2
...
+  image: kubeshop/testkube-cypress-executor:1.1.7-cypress10   # <-- we're buidling cypress versions
+  types:
+  - cypress:v10/test # <-- just create different test type with naming convention "framework:version/type"
+```
+
+> Tip: Look for recent executor versions here https://hub.docker.com/repository/registry-1.docker.io/kubeshop/testkube-cypress-executor/tags?page=1&ordering=last_updated.
+
+
+And add it to your cluster: 
+```sh
```
Comment (fivenp) [inline docs/docs/4-test-types/executor-cypress.md:307]:
```suggestion
```bash
```

File: docs/docs/4-test-types/executor-cypress.md [hunk 6]
```diff
@@ -277,6 +278,46 @@ $ kubectl testkube get execution 615d5372b046f8fbd3d955d2
...
+kubectl apply -f cypress-v10-executor.yaml 
+```
+
+Now, create a new test with a type which our new executor can handle e.g.: `cypress:v10/test`
+
+```sh
```
Comment (fivenp) [inline docs/docs/4-test-types/executor-cypress.md:313]:
```suggestion
```bash 
```

**Review by fivenp — APPROVED** [review]
LGTM!

**Pull request merged**